source/documentors/references/templates/doc_reference_template.rst

```diff
@@ -0,0 +1,136 @@
+Reference Document Template
+###########################
+
+Reference material is information-oriented. Regarding software, reference guides describe the software - APIs, classes, functions, etc. In the context of the Open edX® Docs, reference docs are often in the form of informational lists (e.g., Resources for Students) and tables (e.g., Course Access Roles and Privileges).
+
+Reference material is like a map. A map tells you what you need to know about the territory without having to go out and check it for yourself; a reference guide serves the same purpose for the product and its internal machinery. For example, this part works as the introduction to the article and as a description of the product of what this document is about. (Replace this text with your content.)
```

Comment:
Full stop should be after the parenthesis:  (Replace this text with your content).

Reply:
```suggestion
Reference material is like a map. A map tells you what you need to know about the territory without having to go out and check it for yourself; a reference guide serves the same purpose for the product and its internal machinery. For example, this part works as the introduction to the article and as a description of the product of what this document is about. 
```

I think "(Replace this text with your content)" is more confusing here than helpful.